### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by ngodfraind**

### CODE REVIEW

File: main/core/Resources/translations/platform.en.yml [hunk 1]
```diff
@@ -101,13 +101,13 @@ answer: Answer
...
 apply: Apply
-apply_recursively_to_directories: 'Apply changes to current and sub directories'
+apply_recursively_to_directories: 'Apply changes to current and subdirectories'
 at_least_one_category_has_to_be_selected: 'At least one category has to be selected'
 attempt: Attempt
 attempts: Attempts
 auth_mode: 'Authentication mode'
 authentication: Authentication
-authentication_invalid: 'The authentication source %authentication% from the line %line% is invalid.'
+authentication_invalid: 'The authentication source %authentication% in line %line% is invalid.'
```
Comment (stefk) [inline main/core/Resources/translations/platform.en.yml]: Should be *at*.

File: main/core/Resources/translations/home.en.yml [hunk 2]
```diff
@@ -2,20 +2,24 @@ Collapse: Collapse
...
 content_size: 'Size of content'
 create_content: 'Create a content'
 days_ago: '%count% day ago|%count% days ago'
-edit_in_page: edit_in_page
+edit_in_page: 'Edit in page'
 for_unlogged_users: 'For not logged users'
+<<<<<<< HEAD
 header: Header
-header_menu_available: 'When header menu is available'
+=======
```
Comment (stefk) [inline main/core/Resources/translations/home.en.yml]: Merge issue.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Pull request merged**